lib/broccoli/ember-app.js

```diff
@@ -242,11 +242,24 @@ EmberApp.prototype._initOptions = function(options, isProduction) {
   @method _initVendorFiles
 */
 EmberApp.prototype._initVendorFiles = function() {
-  // in Ember 1.10 and higher `ember.js` is deprecated in favor of
-  // the more aptly named `ember.debug.js`.
-  var defaultDevelopmentEmber = this.bowerDirectory + '/ember/ember.debug.js';
-  if (!existsSync(path.join(this.project.root, defaultDevelopmentEmber))) {
-    defaultDevelopmentEmber = this.bowerDirectory + '/ember/ember.js';
+  var ember = this.project.findAddonByName('ember-source');
```

Comment:
the discussion in https://github.com/emberjs/ember.js/pull/13022 seemed to have converged on using `ember-core` instead of `ember-source`